doc/classes/Vector3.xml

```diff
@@ -96,7 +96,7 @@
 			<argument index="2" name="post_b" type="Vector3" />
 			<argument index="3" name="weight" type="float" />
 			<description>
-				Performs a cubic interpolation between vectors [code]pre_a[/code], [code]a[/code], [code]b[/code], [code]post_b[/code] ([code]a[/code] is current), by the given amount [code]weight[/code]. [code]weight[/code] is on the range of 0.0 to 1.0, representing the amount of interpolation.
+				Cubically interpolates between this vector and [code]b[/code] using [code]pre_a[/code] and [code]post_b[/code] as handles, and returns the result at position [code]weight[/code]. [code]weight[/code] is on the range of 0.0 to 1.0, representing the amount of interpolation.
```

Comment:
I don't feel strongly about it but I think "Performs a cubic interpolation" was easier to read and would be easier to translate.

Reply:
Ah, I picked the `Vector2` version because it also explains `pre_a` and `post_b` does. I'll change this to use "Performs a cubic interpolation...". After your CI update PR is merged :)

Reply:
Yeah you can keep the rest of the changes, just change the start of the sentence.